packages/roosterjs-editor-plugins/lib/plugins/ImageSelection/ImageSelection.ts

```diff
@@ -0,0 +1,73 @@
+import { safeInstanceOf } from 'roosterjs-editor-dom';
+import {
+    EditorPlugin,
+    IEditor,
+    PluginEvent,
+    PluginEventType,
+    SelectionRangeTypes,
+} from 'roosterjs-editor-types';
+
+/**
+ * Detect image selection and help highlight the image
+ */
+export default class ImageSelection implements EditorPlugin {
+    private editor: IEditor | null = null;
+    private image: HTMLImageElement | null = null;
```

Comment:
do we need to cache the image here? Can we get it (its id) using getSelectionRangeEx()?

Reply:
Instead of saving the image, I'm saving only the id.